From linux-usb-owner@vger.kernel.org  Fri Sep 24 09:42:29 2010
From: yauheni.kaliuta@nokia.com
To: linux-usb@vger.kernel.org
Cc: greg@kroah.com, hans.petter.selasky@stericsson.com
Subject: USB: cdc.h: ncm: fix one more typo
Date: Fri, 24 Sep 2010 09:43:27 +0300
Message-Id: <1285310607-22955-1-git-send-email-yauheni.kaliuta@nokia.com>

From: Yauheni Kaliuta <yauheni.kaliuta@nokia.com>

In usb_cdc_ncm_dpe32 the fields are 32 bit long and according
to usb style (hungarian notation) should be called dwDatagramIndex
and dwDatagramLength (see CDC NCM subclass spec, 3.3.2). Actually,
they were called wDatagramIndex, wDatagramLength.

Signed-off-by: Yauheni Kaliuta <yauheni.kaliuta@nokia.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/cdc.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/include/linux/usb/cdc.h
+++ b/include/linux/usb/cdc.h
@@ -343,8 +343,8 @@ struct usb_cdc_ncm_ndp16 {
 
 /* 32-bit NCM Datagram Pointer Entry */
 struct usb_cdc_ncm_dpe32 {
-	__le32	wDatagramIndex;
-	__le32	wDatagramLength;
+	__le32	dwDatagramIndex;
+	__le32	dwDatagramLength;
 } __attribute__((__packed__));
 
 /* 32-bit NCM Datagram Pointer Table */
